stories/uri_components/Visualization.tsx

```diff
@@ -394,6 +395,23 @@ storiesOf("URI components", module)
             </div>,
         ),
     )
+    .add("GeoPushpinChart example", () =>
```

Comment:
Please comment out temporarily this test case as it's unstable now.
We'll bring it back once [SD-948](https://jira.intgdc.com/browse/SD-948) is resolved.

Just note, this case just covers SD-939, there should be one case (without config.tooltip) for SD-868

Reply:
SD-868 include 2 case: one is the same with SD-939, one relate with arithmetic measure. So I think this case still have config with Tooltiptext.